{{IGEM_TopBar}}
{{MRIIRS_FARIDABAD}}
T--mriirs_faridabad--logo.jpg
<html>
<head>
<title>Contribution</title>
<link rel="shortcut icon" href="/favicon.icon">

<!-- Required meta tags -->
    
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    
     <!-- Bootstrap CSS -->
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
    <link rel="stylesheet" href="main.css">
    <link rel="stylesheet" href="animate.css">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>

</head>


	



<body>
      
    <style type="text/css">
        .custom-bg{
            background-color: #000000;
        background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='100%25' height='100%25' viewBox='0 0 800 800'%3E%3Cg %3E%3Ccircle fill='%23000000' cx='400' cy='400' r='600'/%3E%3Ccircle fill='%23180d1c' cx='400' cy='400' r='500'/%3E%3Ccircle fill='%23261431' cx='400' cy='400' r='400'/%3E%3Ccircle fill='%23351947' cx='400' cy='400' r='300'/%3E%3Ccircle fill='%23451e5e' cx='400' cy='400' r='200'/%3E%3Ccircle fill='%23552277' cx='400' cy='400' r='100'/%3E%3C/g%3E%3C/svg%3E");
        background-attachment: fixed;
        background-size: cover;
        }
        ul{
            list-style-type: none;
        }
        body{
            text-align: center;
        }
        ol{
            list-style-type: none;
        }
        </style>
        <style>
          @import url("https://fonts.googleapis.com/css?family=Poppins:400,500,600,700&display=swap");
          * {
            margin: 0;
            padding: 0;
            user-select: none;
            box-sizing: border-box;
            font-family: "Poppins", sans-serif;
          }
         
          body {
            background-color: #1b1b1b;
            font-family: "Poppins", sans-serif;
          }
          nav {
            background: #1b1b1b;
            z-index:100;
          }
          nav:after {
            content: "";
            clear: both;
            display: table;
          }
          nav .logo {
            float: left;
            color: white;
            font-size: 27px;
            font-weight: 600;
            line-height: 70px;
            padding-left: 60px;
          }
          nav ul {
            float: right;
            margin-right: 40px;
            list-style: none;
            position: relative;
          }
          nav ul li {
            float: left;
            display: inline-block;
            background: #1b1b1b;
            margin: 0 5px;
          }
          nav ul li a {
            color: white;
            line-height: 70px;
            text-decoration: none;
            font-size: 18px;
            padding: 8px 15px;
          }
          nav ul li a:hover {
            color: cyan;
            border-radius: 5px;
            box-shadow: 0 0 5px #33ffff, 0 0 10px #66ffff;
          }
          nav ul ul li a:hover {
            box-shadow: none;
          }
          nav ul ul {
            position: absolute;
            top: 90px;
            border-top: 3px solid cyan;
            opacity: 0;
            visibility: hidden;
            transition: top 0.3s;
          }
          nav ul ul ul {
            border-top: none;
          }
          nav ul li:hover > ul {
            top: 70px;
            opacity: 1;
            visibility: visible;
          }
          nav ul ul li {
            position: relative;
            margin: 0px;
            width: 150px;
            float: none;
            display: list-item;
            border-bottom: 1px solid rgba(0, 0, 0, 0.3);
          }
          nav ul ul li a {
            line-height: 50px;
          }
          nav ul ul ul li {
            position: relative;
            top: -60px;
            left: 150px;
          }
          .show,
          .icon,
          input {
            display: none;
          }
          .fa-plus {
            font-size: 15px;
            margin-left: 40px;
          }
          @media all and (max-width: 968px) {
            nav ul {
              margin-right: 0px;
              float: left;
            }
            nav .logo {
              padding-left: 30px;
              width: 100%;
            }
            .show + a,
            ul {
              display: none;
            }
            nav ul li,
            nav ul ul li {
              display: block;
              width: 100%;
            }
            nav ul li a:hover {
              box-shadow: none;
            }
            .show {
              display: block;
              color: white;
              font-size: 18px;
              padding: 0 20px;
              line-height: 70px;
              cursor: pointer;
            }
            .show:hover {
              color: cyan;
            }
            .icon {
              display: block;
              color: white;
              position: absolute;
              top: 0;
              right: 40px;
              line-height: 70px;
              cursor: pointer;
              font-size: 25px;
            }
            nav ul ul {
              top: 70px;
              border-top: 0px;
              float: none;
              position: static;
              display: none;
              opacity: 1;
              visibility: visible;
            }
            nav ul ul a {
              padding-left: 40px;
            }
            nav ul ul ul a {
              padding-left: 80px;
            }
            nav ul ul ul li {
              position: static;
            }
            [id^="btn"]:checked + ul {
              display: block;
            }
            nav ul ul li {
              border-bottom: 0px;
            }
            span.cancel:before {
              content: "\f00d";
            }
          }
          .content {
            z-index: -1;
            position: absolute;
            top: 50%;
            left: 50%;
            transform: translate(-50%, -50%);
            text-align: center;
          }
          header {
            font-size: 35px;
            font-weight: 600;
            padding: 10px 0;
            z-index:-100;
          }
          p {
            font-size: 30px;
            font-weight: 500;
          }
         
    
        </style>
      </head>
      <body>
        <nav class="navbar" style="margin-top:0">
            <div class="logo"><a href ><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD"><img src="https://2020.igem.org/wiki/images/1/1d/T--mriirs_faridabad--team1Un.png" height="80"></a></div>
            <label for="btn" class="icon">
              <span class="fa fa-bars"></span>
            </label>
            <input type="checkbox" id="btn" />
            <ul>
              <li>
                <label for="btn-1" class="show">Project +</label>
                <a href="#">Project +</a>
                <input type="checkbox" id="btn-1" />
                <ul style="z-index:100;">
                  <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Description">Description</a></li>
                  <li style="z-index:100;"><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Proposed_Implementation" >Proposed Implementation</a></li>
                </ul>
              </li>
              <li>
                <label for="btn-1" class="show">Engineering Success +</label>
                <a href="#">Engineering Success +</a>
                <input type="checkbox" id="btn-1" />
                <ul>
                  <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Design_Summary">Design Summary</a></li>
                  <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Lab_Notebook">Lab Notebook</a></li>
                  <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Experimentation_Protocols">Experimentation Protocol</a></li>
                </ul>
              </li>
              <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Contribution">Contribution</a></li>
              <li>
                <label for="btn-2" class="show">Human Practices +</label>
                <a href="#">Human Practices +</a>
                <input type="checkbox" id="btn-2" />
                <ul>
                  <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Collaborations">Collaborations</a></li>
                  <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Human_Practices"> Intigrated Human Practices</a></li>
                  <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Science_Communication">Science Communication</a></li>
                </ul>
              </li>
              <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Model">Project Modeling</a></li>
              <li>
                <label for="btn-2" class="show">Judging +</label>
                <a href="#">Judging +</a>
                <input type="checkbox" id="btn-2" />
                <ul>
                  <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Awards">Awards</a></li>
                </ul>
              </li>
              <li>
                <label for="btn-2" class="show">Team +</label>
                <a href="#">Team +</a>
                <input type="checkbox" id="btn-2" />
                <ul>
                  <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Team_Members">Team Members</a></li>
                  <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Attributions">Attributions</a></li>
                  <li><a href="https://2020.igem.org/Team:MRIIRS_FARIDABAD/Sponsors">Sponsors</a></li>
                </ul>
              </li>
              
            </ul>
          </nav>

<!-- Required meta tags -->
<body>
        
       

<!--work start-->
<div class="custom-bg">
<div class="home container" style="background-color: white; padding-top: 25px; margin-bottom: 30px;">
    <div class="row mt-5">
    <div class="col-lg-6 mt-5 py-5 pl-5">
    </div>
<section class="IGEM PARTS">
        <div class="container" style="padding:10px 5px; text-align: center;" background color="#011627">
             <h1 style="font-size:80px; text-align: center;" class="py-3">CONTRIBUTION</h1>
            <h2 style="font-size:60px; text-align: center" class="py-3">PARTS</h1>
            <p class="py-3" > 
                
                
                <p>
                    Undermentioned is the detailed description of parts that were used in the biobrick:                  </p>
                
                <ul class="a" style="list-style-type: none;">
                    <li>	<h3>BBa_J23100</h3> 
                       <br> It is a constitutive promoter sequence which is to be used for the continuous expression of ADA-AlkB operon. It is one of the three operons of the ADA regulon. The constitutive promoter is required to enhance the expression of Ada protein which is otherwise quite low in the E.coli cell. (Uphoff et al, 2016) .</br>
                        
                        </li>
                  <br>
                    </br>
                   
                        <li><h3>BBa_J61100</h3>
                        <br>This ribosomal binding site is a part of the Anderson RBS family that is compatible with E.coli and other prokaryotes for general protein expression.</br>
                        </li>
                        <br>
                    </br>
                        <li>	<h3>ADA-AlkB gene optimized sequence (Registered as BBa_K3587000)</h3>
                            <br> It is the optimized gene sequence of one of the three operons of the ADA Regulon. It encodes for the Ada protein which (when methylated as part of Ada response) acts as a transcriptional activator for the ADA-AlkB promoter. This sequence, used in the biobrick, is optimized in order to attune itself with the prefix and suffixed to be used.</br>
                             
                             </li>
                             <br>
                    </br>
                   
                        <li><h3>BBa_B0015</h3>	
                        <br>It is a double terminator consisting of BBa_B0010 and BBa_B0012 which is designed by Reshma Shetty. It is one of the most reliable terminators in iGEM repository and is frequently used.</br>
                        </li>
                        <br>
                    </br>
                    
                   
                        <li>		<h3>ADA-AlkB Promoter + RBS (Registered as BBa_K3587005)</h3> 
                        <br>It is an efficient promoter of ADA regulon with a Ribosomal binding site (RBS). This promoter exhibits about a 10-fold higher affinity for the Ada protein (Landini & Volkert, 1995) than the other promoters of the regulon. The reporter sequence is placed downstream to this promoter. </br>
                        </li>
                        <br>
                    </br>
                    <li>		<h3>BBa_E0040</h3>
                        <br>This GFP sequence from iGEM repository is one of the frequently used reporter gene. </br>
                        </li>
                        <br>
                    </br>
                    
                      
                    
               
                </ul>

 <img src="https://2020.igem.org/wiki/images/a/a0/T--MRIIRS_FARIDABAD--Poster_poster20000000.jpg">     



                <p>
                    Best basic part of this biobrick:                 </p>
                    <ul class="a" style="list-style-type: none;">
                        <li>	<h3>ADA-AlkB gene optimized sequence (Registered as BBa_K3587000):</h3>
                           <br> -This is a basic part registered by our group on iGEM repository. This part basically belongs to the genome of E. coli which forms the DNA repair elements and code for Ada methyltransferase and AlkB dioxygenase. The sequence used is optimized using various tools in order to accommodate for restriction sites compatible with standard assembly.</br>
                        	 -Ada methyltransferase that is involved in a methyl-shift reaction taking place at the lesion formed. The methylated form of this protein is an irreversible suicidal state which is involved in transcriptional activation of the promoters of other operons of ADA regulon (Lindahl et al., 1988). 
                        <br>-E. coli alkB-encoded AlkB (EcAlkB) protein belongs to the dioxygenase superfamily which is involved in oxidative demethylation of NA bases. Additionally, it requires non-heme Fe2+ and cofactors, 2-oxoglutarate (2OG) and oxygen (O2). This reaction type is involved in restoring the native base conformation in the nucleic acids (Aravind & Koonin, 2001).
                    </br>
                            </li>
                      <br>
                        </br>
                <p>
            Best composite part of this biobrick:   
                     </p>   
                     <li><h3>ADA-AlkB Promoter + RBS (Registered as BBa_K3587005)</h3>
                        <br>It is an efficient promoter of ADA regulon with a Ribosomal binding site (RBS). This promoter exhibits about a 10-fold higher affinity for the Ada protein (Landini & Volkert, 1995) than the other promoters of the regulon. The reporter sequence is placed downstream to this promoter. This sequence comprises of 10-15 bp stretches where the protein DNA interaction between methylated Ada and nucleic acid sequence takes place in order to act as a transcriptional activator for the subsequent downstream gene.
                    </br>

                     </li> </ol>

                     <br></br>
<ul style="list-style-type: none;">
<li>
 Uphoff S, Lord ND, Okumus B, Potvin-Trottier L, Sherratt DJ, Paulsson J (2016). "Stochastic activation of a DNA damage response causes cell-to-cell mutation rate variation." Science 351(6277);1094-7. PMID: 26941321 Lee YM et al., 2015. RPM peptide conjugated bioreduciblepolyethylenimine targeting invasive colon cancer. <a href="https://doi.org/10.1016/j.jconrel.2015.01.020" target="_blank">https://doi.org/10.1016/j.jconrel.2015.01.020</a> </li>
<li>
Landini P. Volkert MR. Transcriptional activation of the Escherichia coli adaptive response gene aidB is mediated by binding of methylated Ada protein. Evidence for a new consensus sequence for Ada-binding sites. J Biol Chem. 1995;270:8285–8289.
</li> <li> The DNA-repair protein AlkB, EGL-9, and leprecan define new families of 2-oxoglutarate- and iron-dependent dioxygenases. Aravind L, Koonin EV Genome Biol. 2001; 2(3):RESEARCH0007.
</li> <li> Review Regulation and expression of the adaptive response to alkylating agents. Lindahl T, Sedgwick B, Sekiguchi M, Nakabeppu Y Annu Rev Biochem. 1988; 57():133-57.

</li>

</ul>
                     
<br></br>
<br></br>
<br></br>

                
        </div>
    
    
    </section>




    <section class="Description">
        <div class="container" style="padding-left: 10px; padding-right: 10px; text-align: center;">
            <h2 style="font-size:60px; text-align: center;" class="py-3">Literature</h1>
            <p class="py-3;" style="text-align: center;" > 
                A thorough and extensive literature survey was a key aspect in successfully designing the biobricks. The team came up with the idea of detecting alkylating agents like nitrosamines while reading about the havoc that nitrogen pollution creates. Nonetheless, giving life to this project was not an easy job. A publication from <b> Mielecki and Grzesiuk </b>significantly helped us in the ideation process <b>(Mielecki and Grzesiuk, 2014)</b>. It gave us the idea of using Ada which is a protein synthesised in E.coli as part of the ADA regulon. It acts as an inbuilt chemosensor of DNA lesions formed due to the presence of alkylating agents and is a part of the natural DNA repair mechanism of E. coli.
                
                </p>
<br>
<p class="py-3" > 
                Mew and Pidgey of the technical team came up with an initial design for the biobrick. While doing so, a major concern was to estimate the initial concentration of Ada protein in E.coli as this point significantly influenced the sensitivity and limit of detection of our sensor. For this, a publication from <b> Uphoff et al </b> helped us to determine the initial concentration of Ada and we learnt that it is quite low as only about 0-1 molecule are present initially in the cell <b> (Uphoff et al,  2016)</b> . Hence, in order to resolve this issue, we came up with an idea of placing the ADA-AlkB gene downstream to a constitutive promoter in order to increase the initial concentration of Ada and thereby, increasing the sensitivity of our sensor. 
                
                </p>
<br>

<p class="py-3" > 
                Our second proposition, the Pro Biobrick, that not only detects these alkylating agents but also degrades one specific alkylating nitrosamine, that is NDMA, was influenced by the doctoral thesis of Dr Christopher Sales. The references of degradation genes in the thesis gave us a direction to design a biobrick that serves the two purposes of detection as well as degradation. 
                
                </p>
<br>
<ul>
<li>
<p>
Mielecki, Damian, and Elżbieta Grzesiuk. “Ada response - a strategy for repair of alkylated DNA in bacteria.” FEMS microbiology letters vol. 355,1 (2014): 1-11. doi:10.1111/1574-6968.12462 

</p>
</li>
<li>
<p>
Sales, C. M. (2012). Functional genomics of the bacterial degradation of the emerging water contaminants: 1,4-dioxane and N-nitrosodimethylamine (NDMA). UC Berkeley. ProQuest ID: Sales_berkeley_0028E_12166. Merritt ID: ark:/13030/m5vt1wz3. Retrieved from https://escholarship.org/uc/item/7xq1r0kd
</p>
</li>
<li>
<p>
Uphoff S, Lord ND, Okumus B, Potvin-Trottier L, Sherratt DJ, Paulsson J (2016). "Stochastic activation of a DNA damage response causes cell-to-cell mutation rate variation." Science 351(6277);1094-7. PMID: 26941321
</p>
</li>
</ul>
                                
        </div>
    
    
    </div>
</div>




<!--Footer-->


    <!-- Optional JavaScript -->
    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
  </body>

</body>

</html>
